pkg/controller/delivery/reconciler.go

```diff
@@ -70,6 +70,7 @@ func (r *Reconciler) reconcileDelivery(delivery *v1alpha1.ClusterDelivery) error
 	var missing []string
 	for _, resource := range delivery.Spec.Resources {
 		_, err := r.repo.GetDeliveryClusterTemplate(resource.TemplateRef)
+		// FIXME: should be checking if err IsNotFound
```

Comment:
is this something we want to tackle before merge / write a story?

Reply:
We have an example of this change for supply chain reconciler in this PR. If it's not found, there is no 'Error' to report and it should _not_ return a reconciler error. This is fixed in the PR for delivery-informers.

Reply:
oh, I see - cool! thx!